I/ImageFiltering/Compat.toml

```diff
@@ -27,8 +29,13 @@ Compat = "0.18-2.1"
 julia = "0.6-0.7"
 
 ["0.4-0.5"]
-FFTW = "0.0.3-0.2"
 julia = "0.7-1.3"
 
+["0.4-0.6"]
+FFTW = "0.0.3-0.2"
+
 ["0.5.3-0.5.4"]
 Requires = "0.1-0.5"
+
+["0.6"]
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.